### ISSUE TITLE
Replace WP admin pointers with Gutenberg popovers

### ISSUE THREAD

**Issue by westonruter** [ampproject/amp-wp#1479]
**As a user, I should be able to view tooltips, regardless of my accessibility needs.**

- [ ] AC1: Investigate the level of effort to port the Gutenberg popover to the `amp` plugin.
- [ ] AC2: Replace the existing tooltip with the Gutenberg popover that behaves like a modal and plays more nicely with Assistive Technologies (ATs).

---

Per @GaryJones in https://github.com/Automattic/amp-wp/issues/1400#issuecomment-425732009:

> The work seems to be progressing on using the **legacy** pointers. Did the fact they are **not accessible**, and **discouraged**, get ignored? That they may not support keyboard-only users, or tell screen-reader users what the behaviour is?

And @afercia in https://github.com/Automattic/amp-wp/issues/1400#issuecomment-424232988:

> the legacy core "pointers" aren't accessible. They've been discussed a few times in the last years and their usage generally discouraged. Reading in the comments above:
>
>> It opens when the user hovers over the "?" icon 
>
>how about keyboard users? how about communicating to users the "?" icon opens a sort of modal? I'd rather look at the `popover` implementation in Gutenberg, where a toggle button with a proper aria-label and visible button tooltip opens a floating panel that behaves like sort of a modal.

**Comment by postphotos** [comment]
Thanks @westonruter for breaking this out - I've added some ACs here (cc @alcurrie). 👍

**Comment by westonruter** [comment]
This should be tackled with #2316.

**Comment by jwold** [comment]
Weston > These popups are not the correct way to implement a tooltip from an accessibility perspective. Note that there are likely more considerations for accessibility. The primary focus for now though should be moving over to use the Gutenberg component that's responsible for tooltips (found in the edit post screen). That is part of the refactoring of these screens to be using React components. This will happen as part of that refactoring, and we may want to close this issue to say that it will be done as part of #2316.

Related to #1400.
